### ISSUE TITLE
Current release (0.31.3) fails running on Lambda x86_64 architecture

### ISSUE THREAD

**Issue by danielsharvey** [Umkus/lambda-layer-sharp#28]
Thanks for the project, very helpful.

You've indicated that the current release (0.31.3) is compatible with both Lambda architectures (x86_64 and arm64).

It works for me with arm64 but fails for x86_64.

It will load image metadata OK but attempting to resize and image results in the following error:

```
2023-06-18T01:14:15.963Z	7428ffee-8520-41a5-8cbd-41cac6949031	ERROR	Invoke Error 	
{
    "errorType": "TypeError",
    "errorMessage": "A boolean was expected",
    "stack": [
        "TypeError: A boolean was expected",
        "    at /opt/nodejs/node_modules/sharp/index.js:7200:15",
        "    at new Promise (<anonymous>)",
        "    at Sharp._pipeline (/opt/nodejs/node_modules/sharp/index.js:7199:14)",
        "    at Sharp.toBuffer (/opt/nodejs/node_modules/sharp/index.js:6001:15)",
        "    at Runtime.handler (/var/task/index.js:133056:71)"
    ]
}
```

**Comment by Umkus** [comment]
Can you check with the updated version?

**Comment by danielsharvey** [comment]
Fixed, thank you! The latest (0.32.1) now works with `lambda.Architecture.ARM_64` and `lambda.Architecture.X86_64`.

**Comment by Umkus** [comment]
Good to hear!
